src/jit/gentree.cpp

```diff
@@ -16166,12 +16166,7 @@ CORINFO_CLASS_HANDLE Compiler::gtGetClassHandle(GenTree* tree, bool* isExact, bo
 
             if (fieldHnd != nullptr)
             {
-                CORINFO_CLASS_HANDLE fieldClass   = nullptr;
-                CorInfoType          fieldCorType = info.compCompHnd->getFieldType(fieldHnd, &fieldClass);
-                if (fieldCorType == CORINFO_TYPE_CLASS)
-                {
-                    objClass = fieldClass;
-                }
+                objClass = gtGetFieldClassHandle(fieldHnd, isExact, isNonNull);
```

Comment:
I realize that the `isExact` variable was pre-existing, but it tripped me up here, because it appears to be a `bool` rather than a `bool *`. Might be nice to rename it something like `pIsExact`.